opentracing-servlet-extension/opentracing-servlet-extension/src/main/java/org/zalando/opentracing/servlet/extension/ServiceLoaderSpanDecorator.java

```diff
@@ -0,0 +1,22 @@
+package org.zalando.opentracing.servlet.extension;
+
+import io.opentracing.contrib.web.servlet.filter.ServletFilterSpanDecorator;
+import org.apiguardian.api.API;
+
+import java.util.ServiceLoader;
+
+import static java.util.ServiceLoader.load;
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+import static org.zalando.opentracing.servlet.extension.CompositeSpanDecorator.composite;
+
+/**
+ * @see ServiceLoader
+ */
+@API(status = EXPERIMENTAL)
+public final class ServiceLoaderSpanDecorator extends ForwardingSpanDecorator {
+
+    public ServiceLoaderSpanDecorator() {
+        super(CompositeSpanDecorator.composite(load(ServletFilterSpanDecorator.class)));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'CompositeSpanDecorator.composite' due to existing static import 'org.zalando.opentracing.servlet.extension.CompositeSpanDecorator.composite'](https://app.codacy.com/app/whiskeysierra/opentracing-toolbox/pullRequest?prid=4625669)